charts/gxf/Chart.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: v2
 name: gxf
 description: Generic GXF Helm chart
-version: '1.2.3'
+version: '1.2.4'
 icon: https://artwork.lfenergy.org/projects/grid-exchange-fabric/abbrev/color/grid-exchange-fabric-abbrev-color.png
 maintainers:
   - name: OSGP
     email: gxf@lists.lfenergy.org
     url: https://www.lfenergy.org/projects/gxf/
 dependencies:
 - name: activemq-artemis
-  version: "0.0.5"
+  version: "0.0.6"
```

Comment:
```suggestion
  version: "0.0.7"
```

Reply:
because that's the new version in the activemq chart above

Reply:
The build will fail: https://github.com/OSGP/charts/pull/9/commits/998f4a8a69e5bfbf99b1ca62150b36a277e24747
Version 0.0.7 does not exist in the repository yet. So this should be changed in a next iteration.